src/server/cluster/cluster_slot_migration.cc

```diff
@@ -82,6 +82,31 @@ error_code ClusterSlotMigration::Start(ConnectionContext* cntx) {
   return {};
 }
 
+error_code ClusterSlotMigration::Init(uint32_t sync_id, uint32_t shards_num) {
```

Comment:
Do we still call ClusterSlotMigration::Start ?

Reply:
yes I remove it